axelrod/player.py

```diff
@@ -241,7 +241,8 @@ def clone(self):
         return new_player
 
     def reset(self):
-        """Resets history.
+        """
+        Resets history.
         When creating strategies that create new attributes then this method
         should be re-written (in the inherited class) and should not only reset
         history but also rest all other attributes.
```

Comment:
I think this is a little clumsy and not as clear as it could be. How about:

"""Resets a player to its initial state

This method is called at the beginning of each match (between a pair of players) to reset a player's state to its initial starting point. It ensures that no 'memory' of previous matches is carried forward.

The default method resets a player's history, cooperations, defections and state_distribution. Players which have further attributes need to override this method and ensure those additional attributes are also reset.
"""